src/lib/device/device_manager.cpp

```diff
@@ -321,8 +323,8 @@ shared< VirtualDev > DeviceManager::create_vdev(vdev_parameters&& vparam) {
         auto total_chunk_num_in_pdev =
             static_cast< uint32_t >(vparam.num_chunks * (pdev->data_size() / static_cast< float >(total_type_size)));
 
-        LOGINFO("{} chunks is created on pdev {} for vdev {}", total_chunk_num_in_pdev, pdev->get_devname(),
-                vparam.vdev_name)
+        LOGINFO("{} chunks is created on pdev {} for vdev {}, pdev data size is {}", total_chunk_num_in_pdev,
```

Comment:
can we add an assert to ensure `total_chunk_num_in_pdev <= vparam.num_chunks`

Reply:
sure